packages/components/src/components/button/button.spec.tsx

```diff
@@ -51,11 +51,7 @@ test.describe('DBButton component A11y', () => {
 		page,
 		mount
 	}) => {
-		await mount(
-			<DBButton icon="account" onlyIcon={true}>
-				lorem ipsum
-			</DBButton>
-		);
+		await mount(<DBButton icon="account">lorem ipsum</DBButton>);
```

Comment:
```suggestion
		await mount(<DBButton icon="account" icntxt="true">lorem ipsum</DBButton>);
```

As this is the "icon only" testcase, wouldn't it need that attribute instead of removing it altogether?

Reply:
The default with a icon button is without text. So `icntxt` is false and shouldn't be used

Reply:
A button without a visible text is more problematic regarding accessibility, as an element should always contain a visual textual description e.g. for voice control. That for I would expect that having an icon as an additional part of button with a visible text would be the default, and as this isn't clearly defined within the specification, I would at least interpret the order of the elements within Zeplin as well as within Marketingportal that Icon with text would be the default: https://marketingportal.extranet.deutschebahn.com/marketingportal/Design-Anwendungen/db-ux-design-system-v3/components/action/button#:~:text=Handlungsaufforderung%20zu%20beeintr%C3%A4chtigen.-,Content,-(Default)%20Text

Reply:
changed the default behaviour